openpype/tools/utils/lib.py

```diff
@@ -488,9 +489,17 @@ def get_repre_icons():
         "providers", "resources"
     )
     icons = {}
-    # TODO get from sync module
-    for provider in ['studio', 'local_drive', 'gdrive']:
-        pix_url = "{}/{}.png".format(resource_path, provider)
+    if not os.path.exists(resource_path):
+        print("No icons for Site Sync found")
+        return {}
+
+    for file_name in os.listdir(resource_path):
+        if file_name and not file_name.endswith("png"):
+            continue
+
+        provider, _ = os.path.splitext(file_name)
+
+        pix_url = "{}{}{}".format(resource_path, os.path.sep, file_name)
```

Comment:
```suggestion
        pix_url = os.path.join(resource_path, file_name)
```

Reply:
How do you know that the pix url is related to the provider?

Reply:
Thought I had it mentioned somewhere, I didn't, added basic documentation to README.